src/java/fr/paris/lutece/portal/util/mvc/admin/MVCAdminJspBean.java

```diff
@@ -78,11 +78,11 @@
     private static final String MARK_INFOS = "infos";
     private static final String MARK_WARNINGS = "warnings";
     private static Logger _logger = MVCUtils.getLogger( );
-    private List<ErrorMessage> _listErrors = new ArrayList<>( );
-    private List<ErrorMessage> _listInfos = new ArrayList<>( );
-    private List<ErrorMessage> _listWarnings = new ArrayList<>( );
-    private Controller _controller = getClass( ).getAnnotation( Controller.class );
-    private HttpServletResponse _response;
+    private transient List<ErrorMessage> _listErrors = new ArrayList<>( );
+    private transient List<ErrorMessage> _listInfos = new ArrayList<>( );
+    private transient List<ErrorMessage> _listWarnings = new ArrayList<>( );
+    private transient Controller _controller = getClass( ).getAnnotation( Controller.class );
+    private transient HttpServletResponse _response;
```

Comment:
As before, this does not make the class properly serializable.

Reply:
will be rollbacked